lib/models/os.coffee

```diff
@@ -330,13 +342,25 @@ getOsModel = (deps, opts) ->
 	# 	fs.writeFile('foo/bar/config.json', JSON.stringify(config));
 	# });
 	###
-	exports.getConfig = (nameOrId, callback) ->
+	exports.getConfig = (nameOrId, options = {}, callback) ->
+		defaultOpts =
+			appUpdatePollInterval: 1
+			network: 'ethernet'
+			wifiKey: null
+			wifiSsid: null
+			ip: null
+			gateway: null
+			netmask: '255.255.255.0'
```

Comment:
This default is already configured in `resin-device-config`, so I don't think we should duplicate it here.